go.sum

```diff
@@ -339,6 +337,10 @@ github.com/jhump/protoreflect v1.6.0/go.mod h1:eaTn3RZAmMBcV0fifFvlm6VHNz3wSkYyX
 github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af/go.mod h1:Nht3zPeWKUH0NzdCt2Blrr5ys8VGpn0CEB0cQHVjt7k=
 github.com/joho/godotenv v1.3.0/go.mod h1:7hK45KPybAkOC6peb+G5yklZfMxEjkZhHbwpqxOKXbg=
 github.com/jonboulle/clockwork v0.1.0/go.mod h1:Ii8DK3G1RaLaWxj9trq07+26W01tbo22gdxWY5EU2bo=
+github.com/jpkrohling/thrift v1.13.0-patch1 h1:uOByg7OCxSr5J1awmuF76QFlzDLJr6oV6JiXzZ1SX1A=
+github.com/jpkrohling/thrift v1.13.0-patch1/go.mod h1:AhOHZ22laQ1g5hP51Nu1HYddF9D9yQ6V4ibXVSCHKVs=
+github.com/jpkrohling/thrift v1.13.1 h1:lQ/BNkbItLok7GHjz9htWGEIZ7gw00GCnWD6Zq/95ss=
```

Comment:
Still seems to be pulling in your v1.13.1 version.

Reply:
`go mod tidy` to the rescue :-)